### PR TITLE
Configuring www as Default Domain

### PR DESCRIPTION
**Pull request by diana-lakatos**
Configuring www as the Default Domain tutorial for the Partner Portal section. 
@JJBragg Please review. Thank you.

### CODE REVIEW

File: marketplace_builder/views/pages/tutorials/partner-portal/configuring-www-default-domain.liquid [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+* [Accessing the Partner Portal](/get-started/accessing-partner-portal)
+* [Setting Up a Production Domain](/tutorials/partner-portal/setting-up-production-domain)
+
+## Steps 
+
+Setting up the www version of your domain as the default domain is a three-step process:
+
+1. Verify wildcard SSL certificate
+2. Add hosted zone for www
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/partner-portal/configuring-www-default-domain.liquid]:
```suggestion
2. Add hosted zone for WWW
```

File: marketplace_builder/views/pages/tutorials/partner-portal/configuring-www-default-domain.liquid [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+Click on `Create Hosted zone record`. The `www.example.com` record has been added to the list. 
+
+### Step 3:  
+
+On the Instance's page, go to `Domains`, click on `Add new domain`, and fill on the form:
+
+* **Name:** www.example.com
+* **Use as default:** check to enable
+* **Redirect:** check to enable
+   * **Redirect to:** https://www.example.com.
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/partner-portal/configuring-www-default-domain.liquid]: this dot at the end is intentional?

Comment (diana-lakatos) [reply marketplace_builder/views/pages/tutorials/partner-portal/configuring-www-default-domain.liquid]: Not intentional, thanks for catching that!

**Review by pavelloz — APPROVED** [review]

**Comment by diana-lakatos** [comment]
Updated based on JJ's feedback.

**Pull request merged**